wrappers/services.go

```diff
@@ -377,9 +397,13 @@ func genServiceFile(appInfo *snap.AppInfo) []byte {
 	serviceTemplate := `[Unit]
 # Auto-generated, DO NOT EDIT
 Description=Service for snap application {{.App.Snap.InstanceName}}.{{.App.Name}}
+{{- if .MountUnit }}
 Requires={{.MountUnit}}
```

Comment:
Why does a user systemd service not need the mount unit? We added the `Requires={{.MountUnit}}` so that the service does not start before the snap is mounted. Are user service run after all the mounts? If so, a comment would be nice so that our future selfs will remember this.

Reply:
I don't think there is any way a user unit can depend on or trigger a system unit to start.  The user mode systemd generates transient units for existing mount points, but it wouldn't have any way to cause the mount to occur as a prerequisite.  We're pretty much in the same position as a snap app here.